### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
[test]

File: tests/conftest.py [hunk 1]
```diff
@@ -56,6 +56,17 @@ def unsupported_cve():
...
 
 
+@pytest.fixture
+def unsupported_cve_none_versions():
+    """Unsupported ecosystem CVE fixture.
+
+    Some of the CVE versions in this CVE are "None".
+    """
+    with open('tests/data/unsupported-nvdcve-2.json') as f:
+        data, = json.load(f)['CVE_Items']
```
Comment (tisnik) [inline tests/conftest.py]: why do you use comma here? is the structure different than you expect?

Comment (msrb) [reply tests/conftest.py]: Hmm, let me check. I honestly don't remember. Maybe it is a typo? Copy pasted all over the file? :)

Comment (msrb) [reply tests/conftest.py]: It's an array, always with just one element (in this test data).

**Pull request merged**